docs/src/use/configure/configuration-files-new.md

```diff
@@ -145,6 +147,9 @@ export default [
 
 This configuration specifies that all of the files in the `.config` directory should be ignored. This pattern is added after the default patterns, which are `["**/node_modules/**", ".git/**"]`.
 
+Note that only global `ignores` patterns can ignore or unignore whole directories.
```

Comment:
```suggestion
Note that only global `ignores` patterns can match directories.
```

It might be more specific and less confusing to say "match"? Because non-global `foo/**` does effectively ignore the whole directory although it doesn't match any directories.